src/keras_classification/utils.py

```diff
@@ -28,3 +28,8 @@ def make_dir(path, check_empty=False, force_empty=False, use_dirname=False):
     if check_empty and not is_empty:
         raise ValueError(
             '{} needs to be an empty directory!'.format(directory))
+
+
+def predict(batch, model):
+    # Apply same transform to input as when training.
+    return model.predict(batch / 255.0)
```

Comment:
Should this be configurable? Could be a TODO, but as it currently sits keras classification won't work on non-UByte imagery.

Reply:
Yea, this should be configurable. I think the transform should be done at the level of RV though, and that's a more complicated refactor since I would also need to turn off the transform being done by the TF Object Detection API. I made a TODO in the code and added a task to the backlog.